docs/app/components/navigation-bar-search/navigation-bar-search.component.ts

```diff
@@ -32,7 +33,21 @@ export class NavigationBarSearchComponent {
     private data: ISearchResult[];
     private history: ISearchResult[] = [];
 
-    constructor(private router: Router, private navigation: NavigationService) {
+    private angular = true;
+
+    private ngVersions = Version;
+
+    constructor(private router: Router, private navigation: NavigationService, private versionService: VersionService) {
+
+        // get version
+        this.versionService.version.subscribe((value: Version) => {
+            this.angular = value === this.ngVersions.Angular;
```

Comment:
Remove this, when checking version compare the `this.versionService.version.getValue() === Versions.Angular` instead. Incase we ever wish to extend the number of versions, for example add a CSS version then this wouldn't work just using a boolean